### PR TITLE
Filter openMRS records mapped to practitioners from dropDown in create Practitioner 

### LINKED ISSUE #565: Allow only one Practitioner per OpenSRP user
To do this we can simply filter the dropdown in the New Practitioner form to only show OpenMRS usernames which are not mapped to a OpenSRP practitioner.

Screen Shot 2019-10-28 at 10 26 56 AM

### PR DESCRIPTION
**Pull request by p-netm**
__Changes in this__:
- When creating a practitioner, one can no longer pick an openMRS user who is already linked to an existing practitioner.
 

closes #565

### CODE REVIEW

**Comment by cKellyDesign** [comment]
@p-netm what's the status of this one?

**Comment by p-netm** [comment]
> @p-netm what's the status of this one?

@cKellyDesign  i'll push the tests today

**Comment by moshthepitt** [comment]
Please update the branch

File: src/containers/forms/PractitionerForm/UserIdSelect/tests/index.test.tsx [hunk 1]
```diff
@@ -24,15 +25,69 @@ describe('src/*/forms/userIdSelect', () => {
...
+    expect(selectWrapperOptions.length).toEqual(openMRSUsers.results.length - 2);
+
+    // we then look if the records that are missing are actually
+    // those that we want missing i.e from the dropdown options
+    const optionNames = selectWrapperOptions.map((option: any) => option.label);
+    expect(optionNames.includes('superset-user')).toBeFalsy();
+    expect(optionNames.includes('negonga.zatias')).toBeFalsy();
+  });
+
+  it('calls callback correctly with correctly', async () => {
```
[the remark this answers is not in the document]
Comment (cKellyDesign) [reply src/containers/forms/PractitionerForm/UserIdSelect/tests/index.test.tsx]: `calls callback correctly with correct parameters`

**Comment by p-netm** [comment]
@cKellyDesign @moshthepitt Ready for merge

**Pull request merged**